src/pages/landing/weekend-trading/components/_image-text-switching.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, SectionContainer, Show } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+import { localize } from 'components/localization'
+
+const StyledSection = styled(SectionContainer)`
+    background-color: var(--color-white);
+    padding: 8rem 0 0 0;
+
+    @media ${device.tabletL} {
+        padding: 40px 16px 0;
+    }
+`
+const StyledContainer = styled(Container)`
+    flex-direction: column;
+
+    @media ${device.tabletL} {
+        width: 100%;
+    }
+`
+const Content = styled.div`
+    width: 100%;
+    max-width: 58.8rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-right: ${(props) => props.margin_right};
+
+    ${Text} {
+        font-size: 2.4rem;
+        margin-top: 0.8rem;
+
+        @media ${device.tabletL} {
+            text-align: center;
+            font-size: 16px;
+        }
+    }
+
+    @media ${device.tabletL} {
+        margin: 0 auto;
+    }
+`
+
+const ImageWrapper = styled.div`
+    max-width: 47.1rem;
+    width: 100%;
+    max-height: 30rem;
+    margin-right: ${(props) => props.margin_right};
+
+    @media ${device.tabletL} {
+        margin: 2rem auto;
+    }
+`
+const StyledHeader = styled(Header)`
+    font-size: var(--text-size-l);
+    line-height: 1.25;
+
+    @media ${device.tabletL} {
+        width: 98%;
+        margin-top: 0;
+        text-align: center;
+        font-size: 24px;
+    }
+`
+const StyledText = styled(Text)`
+    font-size: 3.2rem;
+    line-height: 40px;
+    text-align: center;
+    font-weight: 700;
+
+    @media ${device.tabletL} {
+        font-size: 24px;
+        line-height: 30px;
+    }
+`
+const Row = styled.div`
+    flex-direction: ${(props) => props.flex_direction};
+    width: 100%;
+    display: flex;
+    margin-top: 8rem;
+
+    &:first-child {
+        margin-top: 0;
+    }
+
+    @media ${device.tabletL} {
+        flex-direction: column;
+        margin-top: 40px;
+    }
+`
+
+const query = graphql`
+    query {
+        buy_sell: file(relativePath: { eq: "landing/crypto1.png" }) {
+            ...fadeIn
+        }
+        local_currency: file(relativePath: { eq: "landing/crypto2.png" }) {
+            ...fadeIn
+        }
+    }
+`
+const ImageTextSwitching = ({ P2P, reverse, two_title }) => {
+    const data = useStaticQuery(query)
+    return (
+        <StyledSection>
+            <StyledContainer>
+                <StyledText>{localize('Trade the markets that never sleep')}</StyledText>
+
+                {P2P.map((item, index) => {
+                    let is_even = reverse ? (index + 1) % 2 : index % 2
```

Comment:
We have a utility function `isIndexEven` for this.

Reply:
done